docs/pages/preview.md

```diff
@@ -0,0 +1,9 @@
+# Preview the latest XMTP documentation
+
+We’re excited to share a [preview](https://docs-xmtp-org-git-inbox-protocol-docs-ephemerahq.vercel.app/) of the latest XMTP documentation. Soon, these preview docs will replace the content on this site, becoming the new standard reference. 
```

Comment:
Same feeling -- I'm worried this leaves confusion about which docs to use.

It's like: in the current docs world, both sets of docs exist, and therefore need to be differentiated. But in the new docs world, only the new docs exist.